buildroot-external/board/generic-aarch64/kernel.config

```diff
@@ -0,0 +1,147 @@
+CONFIG_EFI_STUB=y
+
```

Comment:
This seems to come from generic-x86-64. Lots in here is quite Intel specific and probably will never be seen on a aarch64 platform.

I'd suggest to go with something like this:

```
CONFIG_EFI_STUB=y

CONFIG_VIRTIO=y
CONFIG_VIRTIO_PCI=y
CONFIG_VIRTIO_NET=y
CONFIG_VIRTIO_BALLOON=m
CONFIG_VIRTIO_INPUT=m
CONFIG_VIRTIO_BLK=y
CONFIG_VIRTIO_BLK_SCSI=y
CONFIG_VIRTIO_CONSOLE=m
CONFIG_VIRTIO_VSOCKETS=m
CONFIG_VIRTIO_MMIO=y
CONFIG_SCSI_VIRTIO=y
CONFIG_HW_RANDOM_VIRTIO=y

CONFIG_DRM_VIRTIO_GPU=m
```